libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -103,6 +105,11 @@ void AC_Circle::init()
 
     // set starting angle from vehicle heading
     init_start_angle(true);
+
+    _attitude_control.get_slew_yaw_max_degs();
+    if(get_rate() > _attitude_control.get_slew_yaw_max_degs()) {
+        gcs().send_text(MAV_SEVERITY_CRITICAL, "CIRCLE_RATE is High");
```

Comment:
It might be better if the mesage is, "Circle: warning CIRCLE_RATE>ATC_RATE_Y_MAX or ATC_SLEW_YAW"